### ISSUE TITLE
Some unexpected results from your released checkpoints.

### ISSUE THREAD

**Issue by SunnyHaze** [wangjk666/Objectformer#5]
Hi authors!

Thanks for your outstanding work.
 

However, after running an evaluation script on the CASIAv1 dataset with your official checkpoint trained with CASIAv2, we found that the F1 score is `0.1918`. After carefully checking the code I can not find any clues about these unexpected results. 

Detailly, I implemented the F1 metrics with the third mask returned from your `nn.Modules` class. 
https://github.com/wangjk666/Objectformer/blob/a967ba91a77e79f6d64e5a02e15f4214197ceb61/ObjectFormer/models/objectformer.py#L379

Here are the visualizing results on CASIAv1:
<img width="1085" alt="6f20237b28ee6d78c8569f5071651ea" src="https://github.com/wangjk666/Objectformer/assets/62173481/033708c6-1c75-4663-9c38-b318d0d808f9">

Do you have any ideas about it? Any help is sincerely appreciated! Thank you very much.

**Comment by wscxwen** [comment]
Friend, can you look at your yaml configuration file and train.txt file, I used CASIAV1 data to import some errors.
